lib/main.js

```diff
@@ -274,23 +265,25 @@ export default {
 
         let match = parseRegex.exec(result);
         while (match !== null) {
-          // Note that these positions are being converted to 0-indexed
-          const line = Number.parseInt(match[1], 10) - 1 || 0;
-          const col = Number.parseInt(match[2], 10) - 1 || undefined;
+          const line = Number.parseInt(match[1], 10);
```

Comment:
Occasionally `flake8` returns messages that have no location, please don't change how these are handled as it gracefully makes them a range encompassing the entire first line of the file. Your changes here would make it so that they end up being "located" at `(-1, -1)`, which will just throw an error.